cfgov/processors/django_calendar_event.py

```diff
@@ -0,0 +1,46 @@
+import sys
+import json
+import os.path
+import requests
+
+import dateutil.parser
+import datetime
+
+
+def posts_at_url(url):
+
+    current_page = 1
+    max_page = sys.maxint
+    count = 20000
+
+    while current_page <= max_page:
+
+        url = os.path.expandvars(url)
+        resp = requests.get(url, params={'page': current_page, 'count': str(count)})
+        results = json.loads(resp.content)
+        current_page += 1
+        max_page = int(results['count']) / count + 1
+        for p in results['results']:
+            yield p
+
+
+def documents(name, url, **kwargs):
```

Comment:
It's part of the "protocol" for sheer processors-- a processor doesn't need to pay attention to it, though.